### PR TITLE
Rpc

### PR DESCRIPTION
**Pull request by robertoleksy**

### CODE REVIEW

**Comment by da2ce7** [comment]
can we please have JSON for Modern C++ Version 2.0.7 as a sub-module?

**Comment by robertoleksy** [comment]
done

File: .gitmodules [hunk 1]
```diff
@@ -18,3 +18,6 @@
 	path = depends/sodiumpp
 	url = https://github.com/yedino/sodiumpp.git
 	branch = tigusoft
+[submodule "depends/json"]
+	path = depends/json
+	url = https://github.com/nlohmann/json
```
Comment (da2ce7) [inline .gitmodules]: Please include branch. in this case it should be:
`branch = develop`

**Comment by da2ce7** [comment]
Please rebase this pull request to not include comment: ef7bc36f98c00e0c7708724eb4c5dd2a6926d3c7

**Comment by tigusoft-vm** [comment]
Merged it at first, but now in fact we unmerged it to fix it regarding above remarks from @da2ce7 .

**Pull request merged**